src/components/Pages/Tasks/AllTasks.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { TABLE_TITLES, TITLES_PAGES, BUTTONS_NAMES, PAGES_KEYS } from '../../../shared/constants';
+import { PageTitle } from '../../PageTitle/PageTitle';
+import { Table } from '../../Table/Table';
+import { getAllFakeTasks } from '../../../mockApi/getData';
+import { createTask } from '../../../shared/helpers';
+
+export class AllTasks extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
You use equal state in src/components/Pages/Tasks/Tasks.jsx. You can create helper function called like `initizalizeStartState` and use it instead

Reply:
```suggestion
this.state = initialStateTasks;
```

Reply:
Thanks, fixed.